crates/flux-middle/src/rty/mod.rs

```diff
@@ -771,6 +779,10 @@ impl<T> Binder<T> {
         self.value
     }
 
+    pub fn skip_binder_as_ref(&self) -> &T {
```

Comment:
You can do the same with `.as_ref().skip_binder()`. That's just a couple of extra characters longer, I'm not sure we should ahve the extra method, but also not strongly against it

Reply:
removed.